src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -369,6 +369,9 @@
 helps['network dns zone update'] = """
     type: command
     short-summary: Updates DNS zone properties. Does not modify DNS records within the zone.
+    parameters:
+        - name: --if-match
+          short-summary: Update only if the resource with the same ETAG exists. Use '*' to match any ETAG.
```

Comment:
You could probably leave off the talk about '*' since if you omit --if-match you would expect it to match any (or just ignore it entirely).